modules/rostests/kmtests/ntos_se/SeQueryInfoToken.c

```diff
@@ -240,14 +239,14 @@ START_TEST(SeQueryInfoToken)
     //----------------------------------------------------------------//
 
     AccessState = ExAllocatePool(PagedPool, sizeof(ACCESS_STATE));
-    PsProcessType = ExAllocatePool(PagedPool, sizeof(OBJECT_TYPE));
-    AuxData = ExAllocatePool(PagedPool, 0xC8);
-    GenericMapping = ExAllocatePool(PagedPool, sizeof(GENERIC_MAPPING));
+    // AUX_ACCESS_DATA gets larger in newer Windows version.
+    // This is the largest known size, found in Windows 10/11.
+    AuxData = ExAllocatePool(PagedPool, 0xe0);
```

Comment:
Not using ExAllocatePoolZero or WithTag?

Reply:
Yeah, might as well since I'm already touching it.